civet.dev/reference.md

```diff
@@ -457,6 +457,14 @@ Slices are increasing by default, but you can reverse them with `>` or `>=`:
 reversed := x[..>=]
 </Playground>
 
+If you just want to specify one endpoint of an increasing slice,
+you can avoid `..` altogether:
+
+<Playground>
+x is x[<=i] + x[>i]
```

Comment:
We should remove the `x is` and `+` from these examples because `===` and `+` on slices aren't how one may expect.